.github/workflows/xrayTests.yml

```diff
@@ -19,6 +19,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        suite: [ xray,xsc ]
```

Comment:
```suggestion
        suite: [ xray, xsc ]
```

Reply:
also, this is not doing anything
you should change 
`--test.xray` to be `--test.${{ matrix.suite }}`